### ISSUE TITLE
Hosting Documentation Publicly

### ISSUE THREAD

**Issue by HCLarsen** [crystal-lang/html_builder#14]
This shard has some great documentation, but since it's not part of the core library, it doesn't seem to be hosted anywhere public. Are there any plans to put it up on Github pages or anything?

**Comment by gokg4** [comment]
> This shard has some great documentation, but since it's not part of the core library, it doesn't seem to be hosted anywhere public. Are there any plans to put it up on Github pages or anything?

I agree. I am trying to use this but without a list of examples or documentation it's going to be difficult to use this shard.
